fiftyone/utils/beam.py

```
@@ -338,6 +346,255 @@ def beam_export(
+        save=save,
+        return_outputs=has_reducer,
+        progress=progress,
+    )
+
+    if has_reducer:
+        result_key = f"beam_map_{str(ObjectId())}"
+
+        if reduce_fcn is not None:
+            reduce_cls = reduce_fcn
+        else:
+            reduce_cls = ReduceFcn
```

Ruff (0.8.2)

Use ternary operator `reduce_cls = reduce_fcn if reduce_fcn is not None else ReduceFcn` instead of `if`-`else`-block

Replace `if`-`else`-block with `reduce_cls = reduce_fcn if reduce_fcn is not None else ReduceFcn`

(SIM108)